### ISSUE TITLE
 State changed from starting to crashed

### ISSUE THREAD

**Issue by AmirulAndalib** [TGExplore/Marie-2.0-English#37]
2020-11-11T15:01:18.330792+00:00 heroku[web.1]: Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
2020-11-11T15:01:18.354732+00:00 heroku[web.1]: Stopping process with SIGKILL
2020-11-11T15:01:18.446580+00:00 heroku[web.1]: Process exited with status 137
2020-11-11T15:01:18.490210+00:00 heroku[web.1]: State changed from starting to crashed

how to fix  this i tried to change port var's but still not working bot works for 1 minute and bootloops itself

**Comment by AVATARX0TIC** [comment]
Don't use WEBHOOK use worker

**Comment by AmirulAndalib** [comment]
> Don't use WEBHOOK use worker

do you mean to delete the Var Webhook and add var worker?

**Comment by AVATARX0TIC** [comment]
> > Don't use WEBHOOK use worker
> 
> do you mean to delete the Var Webhook and add var worker?

Yes delete WEBHOOK and port but you don't have to add anything for worker just create Procfile with value *worker: python3 -m tg_bot*
